src/controls/qml/Label.qml

```diff
@@ -18,6 +18,41 @@
 import QtQuick 2.9
 import org.asteroid.controls 1.0
 
+/*!
+   \qmltype Label
+   \inqmlmodule org.controls.asteroid 1.0
+
+   \brief Provides a way to get a text label sized for AsteroidOS.
+
+   This is a simple wrapper for Text that by default has white text with font.pixelSize of \l Dims.defaultFontSize.
+
+   Here is a short example that shows some text in a blue rectangle on a yellow background:
+
+    \qml
+    import QtQuick 2.9
+    import org.asteroid.controls 1.0
+
+    Rectangle {
+        anchors.centerIn: parent
+        anchors.fill: parent
+        color: "yellow"
+        Rectangle {
```

Comment:
It seems to me that we could keep the example more minimal by removing this rectangle ? We could just have the top level rectangle be blue for example

Reply:
Fixed